### ISSUE TITLE
When Executing a Scrape Program returns with a "Failed to Create Process" Error.

### ISSUE THREAD

**Issue by BrianVubao** [rarcega/instagram-scraper#441]
![image](https://user-images.githubusercontent.com/25190751/64829045-8fb3d100-d57f-11e9-8b41-b040093ae147.png)

**Comment by VasiliPupkin256** [comment]
This is not a scraper problem. You have renamed a python folder, removed it from path, installed it from under a different python version. I don't know.

**Comment by BrianVubao** [comment]
> This is not a scraper problem. You have renamed a python folder, removed it from path, installed it from under a different python version. I don't know.

Alright I think I've fixed it. I've updated Python to 7.3.4 and in the Install Location of Python I just created a copy of python.exe and named the copy python3.exe. Every time I run scripts now, I use the python3 command. That subsequently solved the "failed to create processes error" when executying the command "instagram-scraper [PARAMETERS GO HERE]". Thank you for your help and I'm so sorry for not responding. Been using these past 4 days to focus on figuring it out.
